src/Printer/Console.php

```diff
@@ -15,12 +15,12 @@
  */
 class Console implements Printer
 {
-    const LINE_LENGTH = 80;
-    const TAB = 4;
+    const DEFAULT_LINE_LENGTH = 80;
 
     public function printData(OutputInterface $output, FileReportList $fileReportList, HintList $hintList): void
     {
-        $separator = str_repeat('-', self::LINE_LENGTH);
+        $length = (int) (`tput cols` ?: self::DEFAULT_LINE_LENGTH);
```

Comment:
tput cols returns the actual width of the terminal, if its not available a default of 80 will be used